src/test/scala/scalismo/statisticalmodel/StatisticalModelTests.scala

```diff
@@ -139,4 +139,66 @@ class StatisticalModelTests extends ScalismoTestSuite {
 
   }
 
-}
\ No newline at end of file
+}
+
+
+class StatisticalVolumeModelTests extends ScalismoTestSuite {
+
+  implicit val random = Random(42)
+
+  implicit def doubleToFloat(d: Double): Float = d.toFloat
+
+  describe("A statistical Volume mesh model") {
+
+    def compareModels(oldModel: StatisticalVolumeMeshModel, newModel: StatisticalVolumeMeshModel) {
+
+      for (i <- 0 until 10) {
+        val standardNormal = Gaussian(0, 1)(random.breezeRandBasis)
+        val coeffsData = standardNormal.sample(oldModel.rank)
+        val coeffs = DenseVector(coeffsData.toArray)
+        val inst = oldModel.instance(coeffs)
+        val instNew = newModel.instance(coeffs)
+        inst.pointSet.points.zip(instNew.pointSet.points)
+          .foreach {
+            case (pt1, pt2) =>
+              (pt1.toVector - pt2.toVector).norm should be(0.0 +- (0.1))
+          }
+      }
+    }
+
+    it("can be transformed forth and back and yield the same deformations") {
+      val path = getClass.getResource("/TetraMeshModel2.h5").getPath
+      val model = StatismoIO.readStatismoMeshVolumeModel(new File(URLDecoder.decode(path))).get
```

Comment:
In the decode method call please add "UTF-8" as the second argument. The function with just one argument is deprecated.